### PR TITLE
Fixes #9253

### LINKED ISSUE #9253: [FEATURE] wb-postback could trigger events on success or fail for further customization
## Required information

**Is your feature request related to a problem? Please describe.**
The `wb-postback` feature works well when there are static error and success messages to be shown on form submission.  However, there are cases where the error message should be tuned to the specific error from the form's submission, such as using the HTTP response's `statusText` or `responseText` to print the server's message, or perhaps the application would like to add something to the success message, such as thanking the user for registering.

**Describe the solution you'd like**
Two new events could be added that are dispatched before the success/failure message is revealed to allow customization before proceeding to unhide the message.

**Describe alternatives you've considered**
I looked into just having a flag to instruct using the response's `responseText`, but the events provide much more flexibility.

**Who shall do the work?**
I am seeking guidance because I want to contribute

## Additional information (optional)

**Timeline**

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by at88mph**
_(Pick the language of your choice to fill out the following information. / Choisissez la langue de votre choix pour remplir l'information ci-dessous.)_

## What does this pull request (PR) do? / Que fait cette demande « pull » (PR)?
This PR satisfies #9253.  It adds events and triggers them when the form has completed successfully or with a failure, and includes the response for further analysis of the failure or to dynamically provide a message.

## Additional information (optional) / Information additionnelle (optionel)

**General checklist / Liste de contrôle générale**

- [x] Create/update documentation /// Créer/mettre à jour la documentation
- [x] Build and test PR's code /// Rouler le script de compilation et tester le code de la PR
- [x] Ensure documentation is bilingual /// S'assurer que la documentation soit bilingue
  - French documentation was not translated to keep consistent with the page.  The entire page needs translation. 

**Related issues / Requêtes associées**
Closes #9253.

**Screenshots / Captures d'écrans**

### CODE REVIEW

**Review by duboisp — APPROVED** [review]
Thanks for the contribution

File: script/setup [hunk 1]
```diff
@@ -5,7 +5,7 @@ case "$(uname -s)" in
 			npm install -g bower grunt-cli && npm install
 		;;
 	*)
-			if [ $(id -u) != "0" ];
+			if [ $(id -u) == "0" ];
```
Comment (duboisp) [inline script/setup:8]: Can you undo this change, it don't seem related to your PR.

Once that completed, we will merge your PR.

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by at88mph** [comment]
Sure.  Done.

**Review by duboisp — APPROVED** [review]
